manage.py

```diff
@@ -26,6 +26,21 @@ def make_shell_context():
 manager.add_command('db', MigrateCommand)
 
 
+def create_default_admin(first_name, last_name, email, password):
+    if (User.query.filter_by(email=email).first() is not None):
```

Comment:
Nit: parenthesis around statements are not Pythonic

Did flake8 not get mad about this?